pages/contacts.vue

```diff
@@ -1,127 +1,172 @@
-              {{ $t('お問い合わせ内容') }}
+              <t-i18n>{{ $t('お問い合わせ内容') }}</t-i18n>
+            </th>
+            <th class="text-center" scope="col">
+              <t-i18n>{{ $t('局名') }}</t-i18n>
+            </th>
+            <th class="text-center" scope="col">
+              <t-i18n>{{ $t('電話番号') }}</t-i18n>
             </th>
-            <th class="text-center" scope="col">{{ $t('局名') }}</th>
-            <th class="text-center tel" scope="col">{{ $t('電話番号') }}</th>
           </tr>
         </thead>
         <tbody>
           <tr>
-            <td class="content" v-bind="headingAttrs">
-              {{ $t('サイト全般に関すること') }}
+            <th class="content" scope="row" v-bind="headingAttrs">
+              <div>
+                <t-i18n>{{ $t('サイト全般に関すること') }}</t-i18n>
+              </div>
+            </th>
+            <td class="bureau">
+              <t-i18n>{{ $t('政策企画局') }}</t-i18n>
             </td>
-            <td class="bureau">{{ $t('政策企画局') }}</td>
             <td class="tel"><a href="tel:03-5388-2171">03-5388-2171</a></td>
           </tr>
           <tr>
-            <td class="content" v-bind="headingAttrs">
-              {{ $t('サイトの技術面に関すること') }}<br />{{
-                $t('オープンデータ、オープンソースに関すること')
-              }}
+            <th class="content" scope="row" v-bind="headingAttrs">
+              <div>
+                <t-i18n>{{ $t('サイトの技術面に関すること') }}</t-i18n
+                ><br />
+                <t-i18n>{{
+                  $t('オープンデータ、オープンソースに関すること')
+                }}</t-i18n>
+              </div>
+            </th>
+            <td class="bureau">
+              <div>
+                <t-i18n>{{ $t('戦略政策情報推進本部') }}</t-i18n>
+              </div>
             </td>
-            <td class="bureau">{{ $t('戦略政策情報推進本部') }}</td>
             <td class="tel"><a href="tel:03-5320-7930">03-5320-7930</a></td>
           </tr>
           <tr>
-            <td class="content" v-bind="headingAttrs">
-              {{ $t('新型コロナウイルス感染症対策本部会議に関すること')
-              }}<br />{{ $t('都庁来庁者データに関すること') }}
+            <th class="content" scope="row" v-bind="headingAttrs">
+              <div>
+                <t-i18n>{{
+                  $t('新型コロナウイルス感染症対策本部会議に関すること')
+                }}</t-i18n
+                ><br />
+                <t-i18n>{{ $t('都庁来庁者データに関すること') }}</t-i18n>
+              </div>
+            </th>
+            <td class="bureau">
+              <t-i18n>{{ $t('総務局') }}</t-i18n>
             </td>
-            <td class="bureau">{{ $t('総務局') }}</td>
             <td class="tel">
-              {{ $t('感染症対策本部会議に関すること') }}<br /><a
-                href="tel:03-5388-2453"
-                >03-5388-2453</a
-              ><br />
-              {{ $t('都庁来庁者データに関すること') }}<br /><a
-                href="tel:03-5388-2319"
-                >03-5388-2319</a
-              >
+              <t-i18n>{{ $t('感染症対策本部会議に関すること') }}</t-i18n><br />
+              <a href="tel:03-5388-2453">03-5388-2453</a<br />
```

Comment:
`<a>` の閉じタグはこうでしょうか？

```suggestion
              <a href="tel:03-5388-2453">03-5388-2453</a><br />
```

Reply:
失礼しました、Webでのコンフリクト解消ミスでした。